packages/gafl-webapp-service/src/pages/contact/digital-licence/licence-fulfilment/__tests__/result-function.spec.js

```diff
@@ -0,0 +1,58 @@
+import resultFunction from '../result-function.js'
+import { CommonResults, MultibuyForYou } from '../../../../../constants.js'
+
+describe('licence-for > result-function', () => {
+  const mockStatusCacheGet = jest.fn()
+  const mockTransactionCacheGet = jest.fn()
+
+  const mockRequest = {
+    cache: () => ({
+      helpers: {
+        status: {
+          getCurrentPermission: mockStatusCacheGet
+        },
+        transaction: {
+          get: mockTransactionCacheGet
+        }
+      }
+    })
+  }
```

Comment:
You should mock out isMultibuyForYou rather than the request, otherwise you're testing isMultibuyForYou again